app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
Okay, so in terms of the federation:
`sharing` - I send limited posts and profile to the contact
`following` - I request contact to send me public posts
`receiving` - Contact sends me limited posts and profile (when `subscribed` is implemented that is only needed for the green check mark on profile view page actually)
`subscribed` - I send public posts to the contact

Actually this vocabulary feels quite confusing. It's not clear who is `following` who and who is `subscribed` for what out of this words. `following` and `subscribed` are synonyms and if we just swap these two names it won't look like anything changed at all.

How about using the word `followed` instead of `subscribed`? It's better because:
1) It's obvious that it's an opposite of `following` this way
2) It gives grammatical hint: "I am following" -> I read public posts. "I am followed" -> I send public posts

So then we have a simple relation:
`sharing` - I send private messages
`receiving` - I receive private messages
`followed` - I send public messages (I am followed)
`following` - I receive public message